server/api/index.js

```diff
@@ -264,4 +265,48 @@ router.get("/users", (req, res) => {
   });
 });
 
+router.get("/user-roles", async (req, res) => {
+  const data = await db.getUsersByRole();
```

Comment:
I am not sure what this endpoints returns? And why it can't be done in SQL? can you clarify please ... add comment to the code anyhow, as most people won't be familiar with reduce

Reply:
ok .. i see what it returns. Definitely clarify this in a code comment!

Reply:
OK